File: src/main/java/client/database/data/FamilyData.java
package client.database.data;

public class FamilyData {
   private int characterId;

   private int familyId;

   private int seniorId;

   private int reputation;

   private int todaysReputation;

   private int totalReputation;

   private int reputationToSenior;

   private String precepts;

   public FamilyData(int characterId, int familyId, int seniorId, int reputation, int todaysReputation, int totalReputation, int reputationToSenior, String precepts) {
      this.characterId = characterId;
      this.familyId = familyId;
      this.seniorId = seniorId;
      this.reputation = reputation;
      this.todaysReputation = todaysReputation;
      this.totalReputation = totalReputation;
      this.reputationToSenior = reputationToSenior;
      this.precepts = precepts;
   }

   public int getCharacterId() {
      return characterId;
   }

   public int getFamilyId() {
      return familyId;
   }

   public int getSeniorId() {
      return seniorId;
   }

   public int getReputation() {
      return reputation;
   }

   public int getTodaysReputation() {
      return todaysReputation;
   }

   public int getTotalReputation() {
      return totalReputation;
   }

   public int getReputationToSenior() {
      return reputationToSenior;
   }

   public String getPrecepts() {
      return precepts;
   }
}

File: engine/engine-model/src/main/scala/client/database/data/FamilyData.scala
package client.database.data

class FamilyData(private var _characterId: Int, private var _familyId: Int, private var _seniorId: Int,
                 private var _reputation: Int, private var _todaysReputation: Int, private var _totalReputation: Int,
                 private var _reputationToSenior: Int, private var _precepts: String) {
  def characterId: Int = _characterId

  def familyId: Int = _familyId

  def seniorId: Int = _seniorId

  def reputation: Int = _reputation

  def todaysReputation: Int = _todaysReputation

  def totalReputation: Int = _totalReputation

  def reputationToSenior: Int = _reputationToSenior

  def precepts: String = _precepts
}
